.circleci/config.yml

```diff
@@ -55,6 +55,95 @@ jobs:
               pytest --cov=data tests
           name: "Running tests"
     working_directory: ~/repo/tracker
+  tracker_integration:
+    docker:
+      - image: "circleci/python:3.6.1"
+      - image: "mongo:3.6.2"
+    steps:
+      - checkout:
+          path: ~/repo
+      - run:
+          command: |
+              python3 -m venv .env
+              . .env/bin/activate
+              pip install -r requirements.txt
+              mkdir -p domain-scan && wget -q -O - https://api.github.com/repos/cds-snc/domain-scan/tarball | tar xz --strip-components=1 -C domain-scan
+              pip install -r domain-scan/requirements.txt
+              pip install -r domain-scan/requirements-scanners.txt
+          name: "Installing dependencies"
+      - run:
+          command: |
+              sudo apt-get install -y apt-transport-https
+              sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2930ADAE8CAF5059EE73BB4B58712A2291FA4AD5
+              echo "deb http://repo.mongodb.org/apt/debian jessie/mongodb-org/3.6 main" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.6.list
+              sudo apt-get update
+              sudo apt-get install -y mongodb-org-tools=3.6.2
+          name: "Installing dependencies"
+      - run:
+          command: |
+              mkdir csv
+              cat > ./csv/owners.csv  << EOF
+              domain,filler,organization_en,organization_fr
+              canada.ca,,Government of Canada,Gouvernement du Canada
```

Comment:
might want to test with a few different orgs/subdomains, to make sure we have tests for any breakages on that respect.

Reply:
Good idea 👍